src/app/talent/talent-list/talent-list.component.ts

```diff
@@ -21,4 +25,25 @@ export class TalentListComponent extends AbstractListComponent<Talent> {
 			relativeTo: this.activatedRoute
 		});
 	}
+
+	public setItemsPerPage(itemsPerPage: number) {
+		this.itemsPerPage = itemsPerPage;
+		this.currentPage = 1;
+
+		this.getPaginated(this.currentPage, this.itemsPerPage, this.sortBy, this.sortOrder);
+	}
+
+	public setCurrentPage(currentPage: number) {
+		this.currentPage = currentPage;
+		
+		this.getPaginated(this.currentPage, this.itemsPerPage, this.sortBy, this.sortOrder);
```

Comment:
why do you have to send all these parameters each time if they're part of this ? can't the method get them by itself ?